scripts/process/as_svg.R

```diff
@@ -1,266 +0,0 @@
-
-#' Sub mustache keys into text
```

Comment:
i thought we still had uncertainty about whether the functions in this file can be used to keep the user experience speedy. if true, we may not be ready to remove this file already, though I know the functions are still on the other branch and in git history. what's your thinking on this now that you've been looking at vizstorm for a day, lindsay?

Reply:
I went the route of saving everything as geojson, and using that within the D3 code. To be honest, I didn't really understand what this was, just thought it was needed when we handrolled SVG. I see what you mean, though - although we haven't experienced any slowness yet (but probably will when we get working with more data). I can revert so that the file remains. I probably got a little heavy-handed with my cleanup.

Reply:
well...this gets at the question of where we draw the line between R and D3 - which is interesting. One thing, for example, is using R for the GIS side of things - I liked the original projection (and the flexibility that we can use whatever makes sense) over the one we'd need for d3.

Reply:
For what it's worth - D3 should be able to do any projection. I'm just struggling with that right now.

Reply:
yes, i think the R-D3 question is still open. makes sense to me to keep this file in here for now and think about how to optimize our use of the tools. i agree with lindsay that d3 projections should give us all the flexibility we'd need, but am also curious to eventually learn how much doing the projection in R might speed us up (or not). this may or may not be a question we answer in the next few weeks.